src/lxc/cgroups/cgfsng.c

```diff
@@ -1254,13 +1254,15 @@ static int cgroup_rmdir(char *dirname)
 
 	if (rmdir(dirname) < 0) {
 		if (!r)
-			WARN("%s: failed to delete %s: %m", __func__, dirname);
+			WARN("%s: failed to delete %s: %s", __func__, dirname,
+			     strerror(errno));
 		r = -1;
 	}
 
 	if (closedir(dir) < 0) {
 		if (!r)
-			WARN("%s: failed to delete %s: %m", __func__, dirname);
+			WARN("%s: failed to delete %s: %s", __func__, dirname,
```

Comment:
Could you remove the `__func__` macro, please? Our logging function will always spew out the function name by design.